pallets/loans/src/benchmarking.rs

```diff
@@ -133,6 +133,9 @@ where
 		)
 		.unwrap();
 
+		// todo(nuno): I guess this is the right place to register AUSD in the asset
```

Comment:
Can you add some context to this `todo` please? E.g. why do we not resolve it in this PR or can we remove it?

Reply:
This has been removed